pds_doi_service/core/outputs/osti_web_parser.py

```diff
@@ -1,5 +1,5 @@
 #
-#  Copyright 2020, by the California Institute of Technology.  ALL RIGHTS
+#  Copyright 2020-21, by the California Institute of Technology.  ALL RIGHTS
```

Comment:
Use an en-dash "–" for numeric ranges. (Yes I am *that* anal retentive 😝 ignore this!)